src/NuGet.ProjectModel/PackageSpec.cs

```diff
@@ -1,29 +1,36 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using System.IO;
+using Newtonsoft.Json.Linq;
 using NuGet.LibraryModel;
 using NuGet.Versioning;
 
 namespace NuGet.ProjectModel
 {
-    public class Project
+    /// <summary>
+    /// Represents the specification of a package that can be built.
+    /// </summary>
+    public class PackageSpec 
     {
-        public const string ProjectFileName = "project.json";
+        public static readonly string PackageSpecFileName = "project.json";
 
-        public Project()
+        public PackageSpec(JObject rawProperties)
         {
+            Scripts = new Dictionary<string, IEnumerable<string>>(StringComparer.OrdinalIgnoreCase);
             TargetFrameworks = new List<TargetFrameworkInformation>();
+            Properties = rawProperties;
         }
 
-        public string ProjectFilePath { get; set; }
+        public string FilePath { get; set; }
 
         public string ProjectDirectory
```

Comment:
Yeah... I wasn't sure the best way to rename it. I may try removing it (and having callers just do the Path operation themselves)